solidity/deploy/23_authorize_relay_maintainer.ts

```diff
@@ -0,0 +1,27 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "setAuthorizationStatus",
+    true
+  )
+
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "authorize",
+    deployer
```

Comment:
@nkuba @pdyraga 
Do we know the address (or addresses) of the accounts that will be maintaining the relay?
For now I just put the deployer himself here.

Reply:
Let's not authorize anyone now. We will authorize later, when setting up the off-chain maintainer.

Reply:
Removed authorization of the deployer account in 8bd7fa56dc286cf22ca36d6dbe7f34d8a9867fa6.
However, I left the requirement that the maintainers must be authorized (see `setAuthorizationStatus`).
The deployer will need to authorize some account in the future, so retargets can be submitted.